plugin-redfish/rfphandler/update_test.go

```diff
@@ -1,3 +1,17 @@
+//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
```

Comment:
this looks like simplupdate test. What has this to do with current story ?

Reply:
License was missing

Reply:
That then should have been a seperate story and PR